composer.json

```diff
@@ -14,14 +14,14 @@
   },
   "require": {
     "php": "^7.1",
-    "symfony/console": "^4.0",
+    "symfony/console": "^4.0|^5.0",
     "symfony/finder": "^4.0",
     "nikic/php-parser": "^4.0",
     "jakub-onderka/php-console-highlighter": "^0.4",
-    "phpunit/php-timer": "^2.0"
+    "phpunit/php-timer": "^2.0|^3.0"
   },
   "require-dev": {
-    "phpunit/phpunit": "^7.0",
+    "phpunit/phpunit": "^7.5.20|^8.5|^9.0",
```

Comment:
How come your using the `|` operator here? Can't we just use the latest version of these packages and then bump up the major release version when this gets merged?

Reply:
different packages have different minimum php requirements .

the purpose of this PR is not to change the minimum php requirement of povils/phpmnd.